distribution/docs/src/main/resources/content/_integrating/_endpoints/csw-endpoint.adoc

```diff
@@ -1728,127 +762,116 @@ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
                          xmlns:dc="http://purl.org/dc/elements/1.1/"
-                         xmlns:ows="http://www.opengis.net/ows"
                          xmlns:dct="http://purl.org/dc/terms/"
                          xmlns:ns9="http://www.w3.org/2001/SMIL20/Language"
-                         xmlns:ns10="http://www.w3.org/2001/XMLSchema-instance"
-                         version="2.0.2"
+                         xmlns:ns10="http://www.w3.org/2001/XMLSchema-instance" version="2.0.2"
                          ns10:schemaLocation="http://www.opengis.net/csw /ogc/csw/2.0.2/CSW-publication.xsd">
     <csw:TransactionSummary>
-        <csw:totalInserted>1</csw:totalInserted>
-        <csw:totalUpdated>0</csw:totalUpdated>
+        <csw:totalInserted>0</csw:totalInserted>
+        <csw:totalUpdated>1</csw:totalUpdated>
         <csw:totalDeleted>0</csw:totalDeleted>
     </csw:TransactionSummary>
-    <csw:InsertResult>
-        <csw:BriefRecord>
-            <dc:identifier>2dbcfba3f3e24e3e8f68c50f5a98a4d1</dc:identifier>
-            <dc:title>Aliquam fermentum purus quis arcu</dc:title>
-            <dc:type>http://purl.org/dc/dcmitype/Text</dc:type>
-            <ows:BoundingBox crs="EPSG:4326">
-                <ows:LowerCorner>-6.171 44.792</ows:LowerCorner>
-                <ows:UpperCorner>-2.228 51.126</ows:UpperCorner>
-            </ows:BoundingBox>
-        </csw:BriefRecord>
-    </csw:InsertResult>
 </csw:TransactionResponse>
 ----
 
-=== Transaction Update Sub-Operation `HTTP POST`
+.Updating Individual Attributes
+Within the `csw:Transaction` element, use the `csw:RecordProperty` to update individual metacard attributes.
 
-The Update sub-operation is a method to specify values used to change existing information in the catalog.
-If individual record property values are specified in the `Update` element, using the `RecordProperty` element, then those individual property values of a catalog record are replaced.
-The `RecordProperty` contains a `Name` and `Value` element.
-The `Name` element is used to specify the name of the record property to be updated.
-The `Value` element contains the value that will be used to update the record in the catalog.
+Use the `Name` element to specify the name of the record property to be updated and set
+the `Value` element to the value to update in the record.
 The values in the `Update` will completely replace those that are already in the record.
 
-Some properties are given default `Value`s if no `Value` is provided.
+[source,xml,linenums]
+----
+<csw:RecordProperty>
+    <csw:Name>title</csw:Name>
+    <csw:Value>Updated Title</csw:Value>
+</csw:RecordProperty>
+----
+
+.Removing Attributes
+To remove a non-required attribute, send the `csw:Name` without a `csw:Value`.
+
+[source,xml,linenums]
+----
+<csw:RecordProperty>
+    <csw:Name>title</csw:Name>
+</csw:RecordProperty>
+----
+
+Required attributes are set to a default `Value` if no `Value` is provided.
```

Comment:
✏️ 🤷‍♂️  May help to make one of those `Value`s surrounded by quotes instead of \`\`, to signify that they are talking about different things (the actual value vs. the Value xml element).

Reply:
yep. fixed.